fala/templates/base.html

```diff
@@ -102,32 +101,38 @@ <h1 class="govuk-heading-xl govuk-!-margin-top-7">
     }
   </script>
   <script src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
-  <script  nonce="{{request.csp_nonce}}">
+  <script nonce="{{request.csp_nonce}}">
     function errorScroll(s,f) {
       document.getElementById(s).scrollIntoView();
       document.getElementById(f).focus();
     }
   </script>
   <script nonce="{{request.csp_nonce}}">
-    const fala_form = document.querySelector('#fala_questions');
-    if (fala_form) {
-      fala_form.addEventListener('submit', function(e){
-
-      // variables for postcode and organisation name
-      const postcode = document.querySelector("#id_postcode").value;
-      const organisation = document.querySelector("#id_name").value;
-
-      // variables for checkbox
-      const checkboxes = document.querySelectorAll('input[type="checkbox"]:checked');
-      const selected = Array.from(checkboxes).map(x => x.value);
-
-      window.dataLayer.push({
-        'event': "formSubmission",
-        'postcode': postcode,
-        'organisation': organisation,
-        'checkbox': selected,
-      });
-    });
-  };
+    function tagManDataLayer() { // Add Data Layer to Google Tag Manager 
```

Comment:
Looking at this, this could go into a JS doc as it isn't needed for inline purposes compared to the tagman init functions.

Reply:
Can you take a look and see if you can get this to work within side an imported JS document instead of inline. We want to try and avoid inline JS as much as we can.

Reply:
I did consider this, but my rationale for keeping this at the bottom, is that we also need to fire this when the 'cookies== allowed'.

I found it simpler to check value & debug at this this level than in the JS where we have to keep rebuilding it.

Reply:
Moved into it's own JS file